test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -98,6 +98,7 @@
     ("aten::hsplit", datetime.date(2021, 11, 20)),
     ("aten::dsplit", datetime.date(2021, 11, 20)),
     ("aten::_convolution_nogroup", datetime.date(9999, 1, 1)),
+    ("aten::linspace", datetime.date(9999, 1, 1)),
```

Comment:
Is this really our pattern now? Doesn't this mean that people will be able to change the signature for linspace in the future and we won't warn them?

Reply:
That's a good point. I think our expectation is that whenever you introduce a BC breaking change, you will need to introduce upgrader for the operator. I think I should do a follow up where we don't warn anything if there is an upgrader entry in the version map.

Reply:
Or just set the date to something more reasonable so it will start warning again in a month.

Reply:
Updated it to March 1st. But I will follow up with a PR shortly to remove this map now that we can handle BC changes